IconButton/IconButton.module.less

```diff
@@ -65,6 +65,12 @@
 			}
 		}
 
+		&.huge {
+			height: @agate-iconbutton-huge-height;
```

Comment:
I know this is "late to the party" for agate, but wondering why `IconButton` variables use "height" for sizing while `Icon` variables use "size"?  Comparing to moonstone, the same descriptor ("height") is used for both.

Reply:
Let's talk about Agate IconButton offline. :) Basically, it just hasn't seen the level of maintinence and tidying that the other components have.